Cue/app/tabs/library/DeckView.js

```diff
@@ -41,18 +57,69 @@ export default class DeckView extends React.Component {
       display: 'icon',
       onPress: () => {this.props.navigator.pop()}
     }
+
+    // TODO This should be broken out into two files
+    let rightItems
+    if (Platform.OS === 'android') {
+      rightItems = [
+        {
+          title: 'Deck Sharing Options',
+          icon: CueIcons.share,
+        },
+        {
+          title: 'Edit',
+          icon: CueIcons.edit
+        }
+      ]
+    } else {
+      rightItems = [
+        {
+          title: 'Add Card',
+          icon: CueIcons.plus,
+          display: 'icon'
+        },
+        {
+          title: 'Edit',
+          display: 'text',
+        }
+      ]
+    }
+
+    let overflowItems
+    if (Platform.OS === 'android') {
+      overflowItems = [
+        {
+          title: 'Show flagged cards only'
+        }
+      ]
+    }
+
```

Comment:
For sure, this is definitely going to be split into `.ios` and `.android` components.